.github/workflows/release.yaml

```diff
@@ -55,3 +55,33 @@ jobs:
 
       - name: Create deb repository
         run: ci/deploy-deb.sh
+
+  # `update-chart-version` creates a new PR for updating the helm chart
+  update-chart-version:
+    needs: deploy-packages
+    runs-on: ubuntu-24.04
```

Comment:
maybe we can make this a variable to be reused across this file? For instance https://github.com/aquasecurity/trivy/pull/8673/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaL19 still uses 22.04

Reply:
@simar7 I'd keep Ubuntu 22.04 in `Deploy rpm/dep packages` for a while.
we should test and update in another PR, because it may affect on installed packages by default, and then deploy scripts can be broken.

Reply:
Sure in that case we should make this to be the same. I'm just afraid that we're running two different versions in the same action workflow, that may run us into unforeseen issues.

Reply:
done https://github.com/aquasecurity/trivy/pull/8673/commits/49f6b1c3d3d0f492d67674b9e65b49bda52b48f4

Reply:
I tried to use common env variable for `runs-on`, but simply it doesn't work:https://github.com/aquasecurity/trivy-test/actions/runs/14262115200

https://stackoverflow.com/questions/72633058/parameterize-runs-on-value-for-github-action